pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -506,3 +510,24 @@ func IsModuleInSpec(modules []cephv1.Module, moduleName string) bool {
 
 	return false
 }
+
+// ApplyMonitoringLabels function adds the name of the resource that manages
+// cephcluster, as a label on the ceph metrics
+func ApplyMonitoringLabels(c *Cluster, serviceMonitor *monitoringv1.ServiceMonitor) {
```

Comment:
thanks for refactoring, just one final nit:
```suggestion
func applyMonitoringLabels(c *Cluster, serviceMonitor *monitoringv1.ServiceMonitor) {
```

Reply:
Thank you for the review. I have updated it according to the recomnendation.